### DISCUSSION TITLE
Mac testing

### DISCUSSION THREAD

**Discussion by Henri-Langenhoven** [michKneisel/obsidian-compile-notes-plugin#4] [General]
Hi! So I tried it out on Mac as requested. It made the Manuscript folder, but didn't make the Manuscript.md file.

**Comment by michKneisel** [comment]
Thank you for testing. By any chance did you change the settings - specifically the folder where it looks for the notes? I should probably set that to default to empty, currently it has a default of notes and looks for the notes in a subdirectory.

**Reply by Henri-Langenhoven** [reply]
Ah awesome I got it to work. My directory structure was indeed wrong. I don't remember what it was because I've tried a bunch of things since, but it's working now, with the parent file containing all the links in my top-most vault directory, and the `notes` folder with all the children nested inside. I'll create a dummy vault and post a screenshot soon.

I did notice a bug though: If the Mac's trash already contains the `Manuscript.md` file, re-running the command deletes the current file from the `Manuscript` directory, but doesn't recreate it, and I get the error "Error: File already exists." If I empty the trash, it works again.
